content/projects/oop-for-data-science/_index.md

```diff
@@ -1,3 +1,35 @@
 ---
 title: OOP for data science
+ready: False
 ---
+
+## Prerequisite
+
+You need to have completed the Multivariate Regression assignment using the salary dataset before doing this assignment ({{% contentlink "projects/data-science-specific/multivariate-linear-regression" %}}). You will use the data and the model you built in that assignment this OOP assignment.
+
+## Assignment
+It is efficient to put machine learning models and other data science techniques into classes so that we can reuse them later and change attributes without changing the code behind these models. Independent concepts can also be put into independent classes: for example, the functioning of a cross-validate class should not affect the functioning of a linear regression class.
+
+1. Create a metrics class that has methods to compute the Mean Squared Error, R-squared and Adjusted R-squared. The class should have the following attributes:
+
+- `x`: 1D or 2D array of features/predictor variables
+- `y`: 1D array containing the outcome variable
+- `model type` (e.g. linear regression)
+
+2. Create a diagnostic_plots class that has methods to plot the (1) residuals versus predicted values and (2) predicted versus observed values.
+
+3. [OPTIONAL] Create a class that takes a linear regression model object as an attribute and has methods for calculating and displaying the following influential values and errors: leverage, Cook's distance, standardised residuals and studentised residuals. This is easiest to do using statsmethods to create the linear regression model object.
+
+4. Use the salary dataset and the model for predicting Salary that you built in the Multivariate Regression assignment. We are going to compare this model's performance to a new model using all the features EXCEPT yearsrank. Create this second multiple regression model now.
```

Comment:
This is confusing, Please rephrase: 
> Use the salary dataset and the model for predicting Salary that you built in the Multivariate Regression assignment.

I'm not sure if this is what's requested but: 
> Use the model you built in the Multivariate Regression project to predict salary in the salary dataset.